lonboard-viz.ipynb

```diff
@@ -0,0 +1,812 @@
+{
```

Comment:
<p>Should the Z value be included in the Geom? this is probably why the lonboard plot doesn't have heights in the Z dimension.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/geotrees-demo/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>